src/main/java/EventsAndCommands/IgnoreCommand.java

```diff
@@ -0,0 +1,13 @@
+package EventsAndCommands;
+
+import java.lang.annotation.*;
+
+/**
+ * IgnoreCommand
```

Comment:
"Commands annotated with this annotation will *not automatically be instantiated and registered*"?